deploy-board/deploy_board/static/js/deploy-board.js

```diff
@@ -66,6 +66,8 @@ function getDefaultPlacement(capacityCreationInfo) {
     //The abstract name of group can be default assigned
     var defaultAssignedGroup = new Set(['us-east-1a','us-east-1c', 'us-east-1d',
         'us-east-1e', 'us-east-2a', 'us-east-2b', 'us-west-2b', 'us-west-2a']);
+    
+    var defaultAssginedGroupForC5Intance = new Set(['us-east-1a', 'us-east-1d', 'us-east-1e']);
```

Comment:
should we add us-east-2*

Reply:
i think we should keep all the 1a, 1d, 1e in both cell

Reply:
update, thanks.